### DISCUSSION TITLE
Hosting on third party domain

### DISCUSSION THREAD

**Discussion by nisheetpatel** [oleeskild/obsidian-digital-garden#286] [Q&A]
I have loved using digital garden to host notes on my website. I'm not familiar with Typescript or web development, so I went for a rather unsophisticated approach that uses an iframe embed of the vercel page on my website - ([here's my current implementation](https://www.nisheetpatel.me/resources)). The reason I went for this approach is to be able to allow for banner images on top of the page and a navigable menu for the website.

This approach has a couple of unintended consequences:

1. The url for the page on my website doesn't update with each note (because it's in an iframe)
2. The scrolling is a little messed up because there's one for the main website and one for the iframe within

Do you have any suggestions to overcome these? Could you perhaps point me in the right direction for my use case? 

P.S.:
Thanks for all the love and effort that you all put into making Digital Garden so frictionless and user-friendly!

**Comment by mounta11n** [comment]
nice website!

in vercel you can add your custom domain under settings. you have to put a TXT record at your domain provider (you will get the value of the record from vercel once you saved your domain there).
